pynamodb/connection/base.py

```diff
@@ -427,11 +427,19 @@ def _make_api_call(self, operation_name, operation_kwargs):
                     'request_id': headers.get('x-amzn-RequestId')
                 }
 
-                if 'RequestItems' in operation_kwargs:
+                if REQUEST_ITEMS in operation_kwargs:
                     # Batch operations can hit multiple tables, report them comma separated
-                    verbose_properties['table_name'] = ','.join(operation_kwargs['RequestItems'])
+                    verbose_properties['table_name'] = ','.join(operation_kwargs[REQUEST_ITEMS])
+                elif TRANSACT_ITEMS in operation_kwargs:
+                    # Transactional operations can also hit multiple tables, or have multiple updates within
+                    # the same table
+                    table_names = []
+                    for item in operation_kwargs[TRANSACT_ITEMS]:
```

Comment:
Should we de-duplicate table names here, or is there a desire to see them ordered corresponding to the items in the transaction?

Reply:
This seems okay. A little redundant when working with a single table, but I can definitely see it being useful for multi-table